guides/common/modules/ref_webhooks-available-events.adoc

```diff
@@ -4,44 +4,46 @@
 endif::[]
-|Actions Remote Execution Run Host Job Ansible Run Playbook Succeeded |Run an Ansible playbook against given hosts.
-|Actions Remote Execution Run Host Job Ansible Enable Web Console Succeeded |Run an Ansible playbook to enable the web console on given hosts.
-|Actions Remote Execution Run Host Job Puppet Run Host Succeeded |Perform a single Puppet run.
-|Actions Remote Execution Run Host Job Katello Module Stream Action Succeeded |Perform a module stream action using the Katello interface.
-|Actions Remote Execution Run Host Job Leapp Pre-upgrade Succeeded |Upgradeability check for RHEL 7 host.
-|Actions Remote Execution Run Host Job Leapp Remediation Plan Succeeded |Run Remediation plan with Leapp.
-|Actions Remote Execution Run Host Job Leapp Upgrade Succeeded |Run Leapp upgrade job for RHEL 7 host.
-|Build Entered |A host entered the build mode.
-|Build Exited |A host build mode was canceled, either it was successfully provisioned or the user canceled the build manually.
-|Content View Created/Updated/Destroyed |Common database operations on a content view.
-|Domain Created/Updated/Destroyed |Common database operations on a domain.
-|Host Created/Updated/Destroyed |Common database operations on a host.
-|Hostgroup Created/Updated/Destroyed |Common database operations on a hostgroup.
-|Model Created/Updated/Destroyed |Common database operations on a model.
-|Status Changed |Global host status of a host changed.
-|Subnet Created/Updated/Destroyed |Common database operations on a subnet.
-|Template Render Performed |A report template was rendered.
-|User Created/Updated/Destroyed |Common database operations on a user.
+|Actions Remote Execution Run Host Job Ansible Run Playbook Succeeded |Run an Ansible playbook against given hosts.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Ansible Enable Web Console Succeeded |Run an Ansible playbook to enable the web console on given hosts.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Puppet Run Host Succeeded |Perform a single Puppet run.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Katello Module Stream Action Succeeded |Perform a module stream action using the Katello interface.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Leapp Pre-upgrade Succeeded |Upgradeability check for RHEL 7 host.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Leapp Remediation Plan Succeeded |Run Remediation plan with Leapp.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Leapp Upgrade Succeeded |Run Leapp upgrade job for RHEL 7 host.|Actions::RemoteExecution::RunHostJob
+|Build Entered |A host entered the build mode.|Custom event: @payload.id (host id), @payload.hostname (host name).
```

Comment:
@lzap, I'm so sorry I'm late to the party, but `@payload` is not an object you can call different methods dynamicly. It's a simple `Hash` object. The `@payload` can contain `:object` key, under which is actual Active Record object will be stored, on which you can call those methods. Just for simplicily this same object is available under `@object` variable if present.

So, just to summarize: 
 - `@payload` is a `Hash` object, to retrieve the info from it, one must use e.g. `@palyoad[:hostname]`
 - `@object` is an Active Record object, which can understand its own methods, thus one can call e.g. `name` on it directly: `@object.name`. The same object is available in `@payload[:object]`, so `@object.name` is the same as `@payload[:object].name`.

Reply:
@ofedoren since this is merged, could you open a PR please?

Reply:
My bad, I will fix this.

Reply:
https://github.com/theforeman/foreman-documentation/pull/951